pages/index.tsx

```diff
@@ -1,9 +1,12 @@
-import { get } from 'ronin';
-import Link from '@components/link';
+import { use } from '@ronin/blade/server/hooks';
+
+import Link from '@components/link.client';
 import QuotationMark from '@components/quotation-mark';
 
-const Index = async () => {
-  const menuItems = await get.menuItems.orderedBy.ascending(['ronin.updatedAt']);
+import type { MenuItems } from 'ronin';
+
+const Index = () => {
+  const menuItems = use.menuItems.orderedBy.ascending(['ronin.updatedAt']) as unknown as MenuItems;
```

Comment:
Currently `use` hooks are not typed & need to be both updated to remove the `Promise` wrapper. Then in future also update it so types work automatically.
```ts
use.posts<Posts>();
```